docs/GettingStartedDocs/install_oe_sdk-Windows.md

```diff
@@ -41,10 +41,12 @@ C:\Program Files\Git\mingw64\bin\openssl.exe
 
 ### Clang
 
-Download [Clang/LLVM for Windows 64-bit](http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe).
+Download [Clang/LLVM 7 for Windows 64-bit](http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe).
```

Comment:
Line 48 says it supports "or above" and can be downloaded from the link there.  So is line 44 still required?  If so, why is installing 7 required when you want to use a later version?